src/UI/HTMLTooltip.js

```diff
@@ -24,8 +24,18 @@ import { CSS_STYLES } from './styles/styles.js'
 /** @type {HTMLTooltipOptions} */
 export const defaultOptions = {
     wrapperClass: '',
-    tooltipPositionClass: (top, left) => `.tooltip {transform: translate(${left}px, ${top}px);}`,
-    caretPositionClass: (top, left) => `.tooltip--email__caret {transform: translate(${left}px, ${top}px);}`,
+    tooltipPositionClass: (top, left) => `
+        .tooltip {
+            transform: translate(${left}px, ${top}px);
+        }
+    `,
+    caretPositionClass: (top, left, isAboveInput) => `
+        .tooltip--email__caret {
+            ${isAboveInput
+        ? `transform: translate(${left}px, ${top}px) rotate(180deg); transform-origin: 16px;`
+        : `transform: translate(${left}px, ${top}px);`
```

Comment:
@GioSensation This is the conditional styling we were chatting might make sense as toggled by a class. With the `rotate` call being part of the `transform`, I'm not sure we can -- but let me know if you think there's a change we can make here.